port/win32/omrsysinfo.c

```diff
@@ -654,6 +654,7 @@ omrsysinfo_get_OS_version(struct OMRPortLibrary *portLibrary)
 			WideCharToMultiByte(OS_ENCODING_CODE_PAGE, OS_ENCODING_WC_FLAGS, versionInfo.szCSDVersion, -1, &buffer[position], (int)(len - position - 1), NULL, NULL);
 		}
 		PPG_si_osVersion = buffer;
+		PPG_si_osVersionOnHeap = buffer;
```

Comment:
**Note:** `PPG_si_osVersionOnHeap` is implicitly initialized to `NULL` at
https://github.com/eclipse/omr/blob/2388c8e260b7182a3ee2dc97dd31d9ce362ac7b3/port/win32/omrmem.c#L79-L83